src/styles/blocks/nav.scss

```
@@ -0,0 +1,55 @@
+.nav {
+  font-weight: 500;
+
+  &__list {
+    display: flex;
+    list-style-type: none;
+    padding-left: 0;
+    margin: 0;
+  }
+
+  &__item {
+    margin: 0 5px;
+    padding: 0 5px;
+    position: relative;
+
+    &:first-child {
+      padding-left: 0;
+      margin-left: 0;
+    }
+
+    &:last-child {
+      padding-right: 0;
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: inline-block;
+    height: 60px;
+    line-height: 60px;
+    color: $primary-text-color;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 12px;
+
+    &:hover {
+      color: $accent-color;
+    }
+  }
+}
+
+.is-active {
+  color: $accent-color;
+
+  &::after {
+    content: "";
+    position: absolute;
+    top: 56px;
```

you could use bottom instead
```suggestion
    bottom: 0;
```

---

Ok, changed.